docs/src/modules/ROOT/pages/constraints-and-score/performance.adoc

```diff
@@ -4,35 +4,44 @@
 [#scoreCalculationPerformanceTricksOverview]
 == Overview
 
-The `Solver` will normally spend most of its execution time running score calculation,
+The `Solver` will normally spend most of its execution time evaluating moves and running score calculation,
 which is called in its deepest loops.
-Faster score calculation will return the same solution in less time with the same algorithm,
+Faster move evaluation will return the same solution in less time with the same algorithm,
 which normally means a better solution in equal time.
 
 
-[#scoreCalculationSpeed]
-== Score calculation speed
+[#moveEvaluationSpeed]
+== Move evaluation speed
 
-After solving a problem, the `Solver` will log the __score calculation speed per second__.
-This is a good measurement of Score calculation performance,
+After solving a problem, the `Solver` will log the __move evaluation speed per second__.
+This is a good measurement of Move evaluation performance,
 despite that it is affected by non-score calculation execution time.
 It depends on the problem scale of the problem dataset.
 Normally, even for large scale problems, it is higher than ``1000``,
 except if you are using xref:constraints-and-score/score-calculation.adoc#easyScoreCalculation[``EasyScoreCalculator``].
 
 [IMPORTANT]
 ====
-When improving your score calculation, focus on maximizing the score calculation speed,
+When improving your move evaluation, focus on maximizing the move evaluation speed,
```

Comment:
See, I don't agree with this statement.

To improve your performance, you have to increase your score calculation speed - if you focus on move evaluation speed, it means you start removing slow moves, and that is counter-productive. You need to increase the score calculation speed while _keeping_ the existing moves.